bucket/rtlutility.json

```
@@ -0,0 +1,49 @@
+{
+    "version": "1.0.3",
+    "description": "RTL Utility is a tool for measuring the Round Trip Latency of your Digital Audio Workstation (DAW) and audio interface",
+    "homepage": "https://oblique-audio.com/rtl-utility.php",
+    "license": "Freeware",
+    "suggest": {
+        "vcredist2015": "extras/vcredist2015",
+        "vcredist2017": "extras/vcredist2017",
+        "vcredist2019": "extras/vcredist2019"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://oblique-audio.com/downloads/RTL_Utility_1_0_3_x64.exe",
+            "hash": "5764c78d4e7150d187dfd9312b21a77c4d4cbecf54586f83cea7f83ae6918a2b"
+        },
+        "32bit": {
+            "url": "https://oblique-audio.com/downloads/RTL_Utility_1_0_3_Win32.exe",
```

```suggestion
        "64bit": {
            "url": "https://oblique-audio.com/downloads/RTL_Utility_1_0_3_x64.exe#/RTL_Utility.exe",
            "hash": "5764c78d4e7150d187dfd9312b21a77c4d4cbecf54586f83cea7f83ae6918a2b"
        },
        "32bit": {
            "url": "https://oblique-audio.com/downloads/RTL_Utility_1_0_3_Win32.exe#/RTL_Utility.exe",
```
We can change the name of the downloaded file with this syntax, avoids any renaming in pre_/post_install!